metricbeat/docs/modules/elasticsearch.asciidoc

```diff
@@ -5,9 +5,14 @@ This file is generated! See scripts/docs_collector.py
 [[metricbeat-module-elasticsearch]]
 == Elasticsearch module
 
-The Elasticsearch module contains a minimal set of metrics to enable monitoring of Elasticsearch across multiple versions. To monitor more Elasticsearch metrics, use our {stack-ov}/xpack-monitoring.html[monitoring] feature.
-
-The default metricsets are `node` and `node_stats`.
+The Elasticsearch module contains a minimal set of metrics to enable monitoring
+of Elasticsearch across multiple versions. The Elasticsearch X-Pack module
+enables you to monitor more Elasticsearch metrics with our {stack-ov}/xpack-monitoring.html[monitoring] feature.
+
+The default metricsets in the Elasticsearch module are `node` and `node_stats`.
+The default metricsets in the Elasticsearch X-Pack module are `ccr`,
+`cluster_stats`, `index`, `index_recovery`, `index_summary`, `ml_job`,
+`node_stats`, and `shard`.
```

Comment:
I think "default" for the x-pack variant of the Elasticsearch module might be a bit misleading. For this variant to work these must be **exactly** the metricsets that must be specified, no less no more.

Same goes for other product modules' x-pack variants as well.

Reply:
To be clear, this isn't an issue with documentation. It's actually pointing to a smell in the design of the product here. If we wanted such strictness with the x-pack variants for stack modules we shouldn't have allowed users to configure any metricsets when `xpack.enabled: true` was set OR at least checked that the right metricsets were configured when this flag was set. 

I've created https://github.com/elastic/beats/issues/12327 to investigate this.

Reply:
I think there are some cases when you *do* need to specify less than what's in the default configuration file. See https://github.com/elastic/beats/issues/12327#issuecomment-496978772

Reply:
There's discussion happening in #12327 but I don't want that to hold up this PR. Let's go with "default" over here for now. We can always come back and change it later if necessary.